index.bs

```diff
@@ -349,8 +355,8 @@ interface mixin DatagramTransport {
 
 # `WebTransport` Interface #  {#web-transport}
 
-`WebTransport` provides a unified interface to all client-server transports
-that are supported by the WebTransport API.  It implements all of the transport
+`WebTransport` provides an interface to the HTTP/3 transport
+supported by the WebTransport API.  It implements all of the transport
```

Comment:
This sentence was a bit circular already, but seems even more so now. How about:
```suggestion
`WebTransport` provides an API to the HTTP/3 transport functionality
defined in [[!WEB-TRANSPORT-HTTP3]].  It implements all of the transport
```

Reply:
Fixed.